### PR TITLE
Partial refactor, multi region, version tag

### PR DESCRIPTION
**Pull request by elbuo8**

### CODE REVIEW

**Comment by motdotla** [comment]
I like to test for package version, because it forces me to make sure I do bump the version. It acts as sort of a second check.

File: bin/node-lambda [hunk 1]
```diff
@@ -1,26 +1,25 @@
...
-AWS_REGION = process.env.AWS_REGION || "us-east-1",
-AWS_FUNCTION_NAME = process.env.AWS_FUNCTION_NAME || packageJson.name,
-AWS_HANDLER = process.env.AWS_HANDLER || "index.handler",
-AWS_MODE = "event",
-AWS_ROLE = process.env.AWS_ROLE || "missing",
-AWS_MEMORY_SIZE = process.env.AWS_MEMORY_SIZE || 128,
-AWS_TIMEOUT = process.env.AWS_TIMEOUT || 3,
-AWS_DESCRIPTION = process.env.AWS_DESCRIPTION || "",
-AWS_RUNTIME = process.env.AWS_RUNTIME || "nodejs";
+var dotenv = require('dotenv');
```
Comment (motdotla) [inline bin/node-lambda]: nice. airbnb style.

File: lib/main.js [hunk 2]
```diff
@@ -1,88 +1,72 @@
...
-  eventexamplefile = __dirname + "/event.json.example";
-
-  if ( !fs.existsSync( eventfile ) ) {
-    console.log( "event.json file successfully created" )
-    fs.writeFileSync( eventfile, fs.readFileSync( eventexamplefile ) );
-  }
-
+Lambda.prototype.setup = function() {
+  console.log('Running setup.');
+  this._createSampleFile('.env');
```
Comment (motdotla) [inline lib/main.js]: nice refactor

**Comment by motdotla** [comment]
:+1:

**Comment by elbuo8** [comment]
Last push makes sure we stick to single quotes.

**Pull request merged**